tailwind.config.js

```diff
@@ -114,6 +114,10 @@ module.exports = {
       "xxs": "9rem",
       "xs-1": "250px",
       ...defaultTheme.maxWidth,
+      "3/5": "80%"
+    },
+    minWidth: {
+      "1/2": "50%"
```

Comment:
why was this added to the tailwind config? and not e.g. a scss class for the needed logic?

Reply:
We can use this property for all modals, so I assumed it will be better if we have it as a 'class' instead of writing it as a style.